lib/jsdom/browser/Window.js

```diff
@@ -661,6 +661,11 @@ function Window(options) {
     const { forEach } = Array.prototype;
     const { style } = elt;
 
+    // Browsing-context connected, per https://github.com/whatwg/html/issues/4547
+    if (!elt.isConnected || !elt._ownerDocument._defaultView) {
+      return declaration;
+    }
```

Comment:
Can you be sure to test this change? In particular the linked issue isn't about the element being connected; it's about the `<style>` element being connected. Probably the right link is https://drafts.csswg.org/cssom/#dom-window-getcomputedstyle step 5?